package.json

```diff
@@ -18,7 +18,8 @@
     "hapi": "^8.0.0",
     "hoek": "^2.6.0",
     "swaggerize-builder": "^2.0.0",
-    "core-util-is": "^1.0.1"
+    "core-util-is": "^1.0.1",
```

Comment:
ok, tried to fix this mess with an online edit but can't get rid of the diff on the core-utils-is